### DISCUSSION TITLE
Obsidian Tasks Showing Up As UUID

### DISCUSSION THREAD

**Discussion by IcedRooibos** [akosbalasko/yarle#485] [General]
I'm trying to import tasks but it shows up as this UUID - is there anything I can do to fix it?
I already have the tasks plugin installed on Obsidian in case that's relevant.

![image](https://github.com/akosbalasko/yarle/assets/47491349/04a16af0-5c68-4bad-9b5e-b3c487bfd574)

**Comment by akosbalasko** [comment]
Hi @IcedRooibos ,
Nope, it's not relevant, this text and code is a placeholder produced by Yarle to identify the real place of the tasks. So these kind of lines should be removed after the conversion. But it's hard to say anything based on just this, but if you send me the log content of the conversion I'd be happy to check it and try to figure the error out. 
here is my email address: akos0215@gmail.com
cheers,
akos

**Reply by IcedRooibos** [reply]
Thank you for that insight @akosbalasko!

I deleted everything and imported a single note as a test case and that worked fine and imported the tasks correctly.

I have now started reimporting everything again. I've just checked and Yarle is open and still using around 13% CPU even though it says import is completed in the logs. I wonder if it's replacing the placeholder tasks with the real values and just taking some time to get through it since I'm importing a bunch of notes (1k+ with lots of embedded attachments).

I will leave things running for some time and report back - maybe it's just taking some time to run through things? If that still doesn't work I am happy to send you the logs.

**Reply by IcedRooibos** [reply]
Geez the issue was that Evernote has corrupted my old note files and the tasks have been deleted...

https://discussion.evernote.com/forums/topic/146605-i-do-not-trust-evernote-anymore-update-issues-loss-of-notestasks-and-syncing-not-working/
https://discussion.evernote.com/forums/topic/146270-note-loading-and-tasks-sync-issues/#comments

Glad I'm moving off.

The problem is resolved, thank you.

**Reply by akosbalasko** [reply]
@IcedRooibos wow... well... auch... Hope you feel yourself outside more safely, I definitely do!

**Reply by IcedRooibos** [reply]
Actually I may have spoke to soon, unfortunately there does indeed seem to be an error. I will email you the logs.
